studio/components/interfaces/ProjectAPIDocs/ProjectAPIDocs.tsx

```diff
@@ -0,0 +1,219 @@
+import { useState, useEffect } from 'react'
+import {
+  Button,
+  CommandGroup_Shadcn_,
+  CommandItem_Shadcn_,
+  CommandList_Shadcn_,
+  Command_Shadcn_,
+  IconTerminal,
+  PopoverContent_Shadcn_,
+  PopoverTrigger_Shadcn_,
+  Popover_Shadcn_,
+  SidePanel,
+} from 'ui'
+
+import { useParams } from 'common'
+import { useProjectApiQuery } from 'data/config/project-api-query'
+import { useAppStateSnapshot } from 'state/app-state'
+import {
+  Bucket,
+  EdgeFunction,
+  EdgeFunctions,
+  Entities,
+  Entity,
+  Introduction,
+  RPC,
+  Realtime,
+  Storage,
+  StoredProcedures,
+  UserManagement,
+} from './Content'
+import FirstLevelNav from './FirstLevelNav'
+import SecondLevelNav from './SecondLevelNav'
+import { useCustomDomainsQuery } from 'data/custom-domains/custom-domains-query'
+import { BASE_PATH } from 'lib/constants'
+
+const ProjectAPIDocs = () => {
+  const { ref } = useParams()
+  const snap = useAppStateSnapshot()
+
+  const { activeDocsSection } = snap
```

Comment:
I'm not sure if destructing is ideal for valtio because it doesn't always work as expected with js proxies. I'd stay on the safe side here and just always do `snap.activeDocsSection`

Reply:
removed this one - realised its not needed